pkg/kudoctl/cmd/init.go

```diff
@@ -148,6 +150,9 @@ func (initCmd *initCmd) run() error {
 	if initCmd.image != "" {
 		opts.Image = initCmd.image
 	}
+	if initCmd.imagePullPolicy != "" {
+		opts.ImagePullPolicy = initCmd.imagePullPolicy
+	}
```

Comment:
Please add some validation here, i.e. "initCmd.imagePullPolicy" must be one of "Always", "IfNotPresent" and "Never"